www_src/pages/login/form-input.jsx

```diff
@@ -3,13 +3,21 @@ var React = require('react');
 var FormInput = React.createClass({
   getDefaultProps: function () {
     return {
-      type: 'text'
+      type: 'text',
+      tabIndex: 0
     };
   },
+  checkForReturn: function (e) {
+    if (e.keyCode === 13) {
+      if (typeof this.props.onReturn === 'function') {
```

Comment:
Not sure I understand. From an ergonomics perspective, a dev can set `onReturn` to any value or type. Because hitting the return button while testing is something that could be easily overlooked, checking to ensure that `onReturn` is a function seems perfectly reasonable.